tests/containers/podman_bci_systemd.pm

```diff
@@ -30,6 +30,9 @@ sub run {
     record_info('Test', 'Start nginx');
     assert_script_run("podman exec nginx systemctl start nginx");
 
+    record_info('Test', 'Wait for container to be healthy');
+    script_retry("podman inspect -f '{{.State.Health.Status}}' nginx | grep -x healthy", retry => 10, delay => 15);
```

Comment:
```suggestion
    script_run("podman healthcheck run");
    script_retry("podman inspect -f '{{.State.Health.Status}}' nginx | grep -x healthy", retry => 10, delay => 30);
```
By default the healthcheck is executed every 30 seconds, so I would use the same frequency for the health check.

Reply:
done and tested